docs/getting-started.md

```diff
@@ -4,7 +4,7 @@ Thanks for your interest in using containerd and Firecracker together!  This
 project is currently in a very early state, and things are not yet as
 plug-and-play as we'd like.  This guide contains general instructions for
 getting started, but is more descriptive than a concrete set of step-by-step
-instructions. 
```

Comment:
My text-editor auto-trim trailing whitespaces on save. Happy to revert it back if needed.